README.md

```diff
@@ -2,7 +2,16 @@
 
 [![Build Status](https://github.com/CMSgov/bcda-app/actions/workflows/ci-workflow.yml/badge.svg?branch=master)](https://github.com/CMSgov/bcda-app/actions?query=branch%3Amaster)
 
-## Dependencies
+## **Documentation**
+
+[API documentation](https://bcda.cms.gov/sandbox/user-guide/)
+
+# 
```

Comment:
is this blank header intentional? 🙂

Reply:
oh that does look odd -- I didn't notice that previously. I'll remove that!